wykorzystanie_formatu_paczki_archiwalnej.rst

```diff
@@ -0,0 +1,96 @@
+Wykorzystanie formatu paczki archiwalnej jako formatu danych EZD
+================================================================
+
+Wstęp
+-----
+
+W tym dokumencie opisana jest propozycja użycia tzw. formatu paczki
+archiwalnej [#paczka_archiwalna]_ jako głównego `formatu danych EZD`_.
```

Comment:
Uściślając plik `Metadane-1.7.xsd` to tylko propozycja formatu paczki archiwalnej. Obowiązująca jest wersja 1.6 (co nie znaczy, że powinna ona stanowić podstawę dalszych prac).